### PR TITLE
Bump dep

### PR DESCRIPTION
**Pull request by kpman**
Using prettier `1.13.5`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,22 +40,22 @@
...
   },
   "dependencies": {
-    "@slack/client": "^4.2.0",
+    "@slack/client": "^4.2.2",
     "axios": "^0.18.0",
-    "body-parser": "^1.18.2",
+    "body-parser": "^1.18.3",
     "camel-case": "^3.0.0",
-    "chalk": "~2.3.2",
+    "chalk": "~2.4.1",
```
Comment (tw0517tw) [inline package.json]: any idea why we used `~` for chalk?

**Comment by tw0517tw** [comment]
the flow error may be related to these issues
https://github.com/facebook/flow/issues/6321
https://github.com/facebook/flow/issues/6181

File: package.json (hunk 1 above)
Comment (chentsulin) [reply package.json]: For old version flowtype, so just use `^` instead.

File: src/context/ConsoleContext.js [hunk 2]
```diff
@@ -38,7 +38,9 @@ export default class ConsoleContext extends Context implements PlatformContext {
       // $FlowExpectedError
       return new Proxy(this, {
         get(target, key) {
+          // $FlowFixMe: Cannot get `target[key]` because an indexer property is missing in `ConsoleContext` [1].
```
Comment (chentsulin) [inline src/context/ConsoleContext.js]: Should we add github issue link reference here?

**Pull request merged**